test/Microsoft.ML.Benchmarks/Numeric/Ranking.cs

```diff
@@ -85,8 +85,11 @@ public void SetupScoringSpeedTests()
             if (!File.Exists(_mslrWeb10k_Train))
                 throw new FileNotFoundException(string.Format(Errors.DatasetNotFound, _mslrWeb10k_Train));
 
+#if !NET461
+            _modelPath_MSLR = Path.Combine(Directory.GetParent(typeof(RankingTest).Assembly.CodeBase.Substring(8)).FullName, @"FastTreeRankingModel.zip");
```

Comment:
`Substring(8)` is very fragile. Can find a way to avoid the substring? Sth like:

```suggestion
            _modelPath_MSLR = Path.Combine(Path.GetDirectoryName(typeof(RankingTest).Assembly.Location), "FastTreeRankingModel.zip");
```